_sass/_py1.scss

```diff
@@ -0,0 +1,374 @@
+    @media (min-width: $desktop-width) {
+	max-width: 25%;
+    }
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.py1-lesson-header {
+    text-decoration: underline 7px $py-blue;
+    text-underline-offset: 22%;
+    margin-bottom: 50px;
+}
+
+.py1-lesson-page-description {
+    // margin-bottom: 50px;
+}
+
+.py1-lesson-page-agenda {
+    margin-top: 50px;
+    margin-bottom: 0px;
+}
+
+.py1-lesson-brief-agenda {
+    margin-top: 0px;
+    margin-bottom: 50px;
+}
+
+.py1-lesson-page-action-bar {
+    // border: solid 1px red;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-right: 5%;
+    margin-bottom: 200px;
+    padding: 0;
+    list-style: none;
+
+    a {
+        background-color: $py-blue;
+        color: white;
+        padding: 5px;
+        padding-right: 30px;
+        padding-left: 30px;
+        border-radius: 10px;
+        font-family: $font-display;
+        text-decoration: none;
+        font-weight: bold;
+        transition: ease-in-out .2s;
+    }
+    
+    a:hover {
+        background-color: $py-blue-tint;
+    }
+}
+
+.py1-lesson-page-divider {
+    margin-left: 50px;
+    width: 1px;
+    background-color: $py-blue;
+}
+
+.py1-lesson-page-next {
+    display: flex;
+    justify-content: center;
+    border: solid 2px $py-blue;
+    background-color: white;
+    margin-bottom: 32px;
+    border-radius: 10px;
+    transition: ease-in-out .2s;
+
+    a {
+        width: 100%;
+        padding: 10px;
+        text-align: center;
+        color: $py-blue;
+        font-family: $font-display;
+        text-decoration: none;
+        font-weight: bold;
+    }
+}
+
+.py1-lesson-page-next:hover {
+    background-color: rgb(232, 232, 232);
+}
+
+.py1-lesson-content {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 40px;
+    margin-left: 50px;
+    @media (max-width: $desktop-width) {
+        width: 100%;
+        margin-left: 0px;
+    }
+    
+    // border: solid 2px $py-blue;
+
+    h1 {
+        width: 100%;
+        text-align: start;
+    }
+
+    iframe {
+        border: solid 2px $py-green;
+        width: 100%;
+        @media (max-width: $desktop-width) {
+            max-width: 480px;
+            height: 300px;
+        }
+        margin-bottom: 50px;
+        
+    }
+}
+
+.py1-lesson-practice {
+    margin-bottom: 32px;
+    padding: 24px;
+    border: $py-green solid 2px;
+    border-radius: 10px;
+
+    a {
+        background-color: $py-green;
+        color: white;
+        padding: 5px;
+        margin-bottom: 20px;
+        padding-right: 30px;
+        padding-left: 30px;
+        border-radius: 10px;
+        font-family: $font-display;
+        text-decoration: none;
+        font-weight: bold;
+        transition: ease-in-out .2s;
+        
+    }
+
+    a:hover {
+        background-color: $py-green-tint;
+    }
+}
```

Comment:
Seems somewhat duplicate with the next lesson button (and maybe the action bar) - consider using a [SASS mixin](https://sass-lang.com/documentation/at-rules/mixin) to help us duplicate less code, or create a base class and then override specific properties.